src/components/layout/inputs/CheckboxInput/CheckboxInput.tsx

```diff
@@ -2,23 +2,35 @@
 * @debt directory "Gaël: this file should be migrated within the new directory structure"
 */
 
-import PropTypes from 'prop-types'
 import React from 'react'
 
+interface IProps {
+  SvgElement?: React.ComponentType | null,
+  checked?: boolean,
+  className?: string,
+  disabled?: boolean,
+  hiddenLabel?: boolean,
+  isInError?: boolean,
+  isLabelDisable?: boolean,
+  label: string,
+  name: string,
+  onChange: () => void,
+  subLabel?: string | null,
```

Comment:
idem, tu n'as pas besoin du null
```suggestion
  subLabel?: string,
```